channel/manager.go

```diff
@@ -19,18 +19,28 @@ package channel
 
 import (
 	"sync"
+	"sync/atomic"
 
 	"github.com/megaease/easeprobe/notify"
 	"github.com/megaease/easeprobe/probe"
 )
 
 var channel = make(map[string]*Channel)
 var wg sync.WaitGroup
-var dryNotify bool
+var dryNotify atomic.Value
+
+func init() {
+	SetDryNotify(false)
+}
 
 // SetDryNotify sets the global dry run flag
 func SetDryNotify(dry bool) {
-	dryNotify = dry
+	dryNotify.Store(dry)
+}
+
+// GetDryNotify returns the dry run flag
+func GetDryNotify() bool {
```

Comment:
I think `IsDryNotify` is a better naming.

Reply:
done!